BaseFilter/sections/specific.txt

```diff
@@ -4254,6 +4254,7 @@ f1livegp.me##.col-sm-6 > div[class="pull-left watch-ll a-li watch"]
 loptelink.com##center > a[href^="https://loptelink.com/ref/"] > img
 loptelink.com##center > .separator > a[style] > img
 f1livegp.me##body a[href^="https://trk.bestconvertor.club/"]
+f1ix.com##.video-holder div:has(> script[src*="//profitablegatecpm.com"])
```

Comment:
I guess it can be changed to one of this rules.
```adblock
f1ix.com##div[style^="height:"]:has(> script[src^="//profitablegatecpm.com/"])
f1ix.com##.col-video > div[style^="height:"]
```
The last one seems to be ok on my end.

Reply:
I like the one proposed by alex
f1ix.com##.col-video > div[style^="height:"]:has(> script)
it looks more targeting

Reply:
According to the code, there it's just a leftover. So, I would not specify it, but the rule with `:has()` is acceptable.